test/java/seedu/addressbook/parser/ParserTest.java

```diff
@@ -175,14 +176,14 @@ public void parse_findCommandInvalidArgs_errorMessage() {
     }
 
     @Test
-    public void parse_findCommandValidArgs_parsedCorrectly() {
+    public void parse_findCommandValidArgs_parsedCorrectlyAsUppercase() {
```

Comment:
Any reason you only renamed this test but not `parse_findCommandDuplicateKeys_parsedCorrectly`?

Reply:
Oops, an oversight on my part.